.pre-commit-config.yaml

```
@@ -24,7 +25,7 @@ repos:
         name: isort (python)
     -   id: isort
```

The two `isort` hooks can be combined into one
```suggestion
    -   id: isort
        types: [text]
        types_or: [python, cython]
```

---

Thank you @MarcoGorelli for the explanation.